frameworks/hdfs/universe/config.json

```
@@ -316,6 +337,27 @@
           "type": "string",
           "description": "Volume profile to be used for storing NameNode data."
         },
+        "external_volume": {
+          "type": "object",
+          "description": "The external volume configuration.\nOnly Portworx external volumes are supported.",
+          "properties": {
+            "enabled": {
+              "type": "boolean",
+              "description": "If true, external profile will be used.",
+              "default": false
+            },
+            "volume_options": {
+              "type": "string",
+              "default": "size=10",
```

```suggestion
              "default": "size=5",
```
To match what Portworx has by default.